.github/workflows/build-extension-charts.yml

```diff
@@ -11,13 +11,26 @@ defaults:
     working-directory: ./
 
 jobs:
+  prepare:
+    runs-on: ubuntu-latest
+    outputs:
+      tagged_release: ${{ steps.release-details.outputs.tagged_release }}
+    steps:
+      - name: Format release details
+        id: release-details
+        # Replace @ with -
+        run: |
+          TAG="${${{ github.ref_name }}//@/-}" 
+          echo "tagged_release=$TAG" >> $GITHUB_OUTPUT
+
   build-extension-charts:
-    uses: rancher/dashboard/.github/workflows/build-extension-charts.yml@master
+    uses: rancher/dashboard/.github/workflows/build-extension-charts.yml@release-2.8
+    needs: prepare
     permissions:
       actions: write
       contents: write
       deployments: write
       pages: write
     with:
       target_branch: gh-pages
-      tagged_release: ${{ github.ref_name }}
+      tagged_release: ${{ needs.prepare.outputs.tagged_release }}
```

Comment:
the `tagged_release` expects `PACKAGE_NAME-VERSION` format but changesets sets `PACKAGE_NAME@VERSION` when runs the release command, so it should be pre-formatted